hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/MockResolver.java

```diff
@@ -397,6 +397,11 @@ public List<String> getMountPoints(String path) throws IOException {
   public void setRouterId(String router) {
   }
 
+  @Override
+  public void rotateCache(
+          String nsId, FederationNamenodeContext namenode, boolean listObserversFirst) {
```

Comment:
Indentation is weird.
Take a look at the checkstyle.

Reply:
done, 8 spaces become 6 spaces,